examples/calculations/submit_singlepoint.py

```diff
@@ -0,0 +1,177 @@
+"""Example code for submitting single point calculation"""
+
+from pathlib import Path
+import sys
+from typing import Union
+
+from ase.build import bulk
+from ase.io import read
+import click
+
+from aiida.common import NotExistent
+from aiida.engine import run_get_node
+from aiida.orm import Code, Str, StructureData, load_node
+from aiida.plugins import CalculationFactory, DataFactory
+
+from aiida_mlip.data.model import ModelData
+
+StructureData = DataFactory("core.structure")
+
+
+def load_model(string: Union[str, None], architecture: str) -> ModelData:
+    """
+    Load a model from a given string.
+
+    If the string represents a file path, the model will be loaded from that path.
+    Otherwise, the model will be downloaded from the specified location.
+
+    Parameters
+    ----------
+    string : Union[str, None]
+        The string representing either a file path or a URL for downloading the model.
+    architecture : str
+        The architecture of the model.
+
+    Returns
+    -------
+    ModelData or None
+        The loaded model if successful, otherwise None.
+    """
+    if string is None:
+        model = None
+        return model
+    file_path = Path(string)
+    print(file_path)
+    if file_path.is_file():
+        model = ModelData.local_file(file_path, architecture=architecture)
+        return model
+    model = ModelData.download(string, architecture=architecture)
```

Comment:
Either want single return or no intermediate variables, personally, I prefer single return. 

**N.B.** for other style replace all `model = ` here with `return`
```suggestion
    if string is None:
        model = None
    elif (file_path := Path(string)).is_file():
        print(file_path)
        model = ModelData.local_file(file_path, architecture=architecture)
    else:
        model = ModelData.download(string, architecture=architecture)
```